assets/src/components/PostTypeSelector/PostTypeSelector.js

```diff
@@ -1,44 +1,43 @@
-import { Component } from '@wordpress/element';
-import { compose } from '@wordpress/compose';
-import { withSelect } from "@wordpress/data";
-import {
-  FormTokenField,
-} from '@wordpress/components';
+import {Component} from '@wordpress/element';
+import {compose} from '@wordpress/compose';
+import {withSelect} from '@wordpress/data';
+import {FormTokenField} from '@wordpress/components';
 
 class PostTypeSelector extends Component {
-  constructor( props ) {
-    super( props );
+  constructor(props) {
+    super(props);
 
-    this.handleChange = this.handleChange.bind( this );
+    this.handleChange = this.handleChange.bind(this);
   }
 
-  handleChange( value ) {
-    const postTypeIds = value.map( postTypeName => {
-      return this.props.postTypes.find( postType => postType.name === postTypeName ).id;
+  handleChange(value) {
+    const postTypeIds = value.map((postTypeName) => {
+      return this.props.postTypes.find((postType) => postType.name === postTypeName).id;
     });
-    this.props.onChange( postTypeIds );
+    this.props.onChange(postTypeIds);
   }
 
   render() {
-    const { postTypes, onChange, label, placeholder, value, ...ownProps } = this.props;
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Maybe you could open a follow-up ticket to refactor this, since it seems it would be doable to just rename the props functions instead of disabling the linting 🤔 But I'm fine with this temporary solution 👍

Reply:
Ok, noted.